awx/ui_next/src/components/DetailList/MissingDetail.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { node } from 'prop-types';
+import styled from 'styled-components';
+import _Detail from './Detail';
+
+const Detail = styled(_Detail)`
+  dd& {
+    color: red;
+  }
+`;
+
+function MissingDetail({ i18n, label }) {
+  return <Detail label={label} value={i18n._(t`Deleted`)} />;
+}
+
+MissingDetail.propTypes = {
+  label: node.isRequired,
+};
+
+export default withI18n()(MissingDetail);
```

Comment:
Since there are other reasons that a detail might be missing (like that piece of data wasn't passed in the response of the api), how would you feel about changing the name here to DeletedDetail?

Reply:
DeletedDetail sounds good to me